great_expectations/analytics/base_event.py

```diff
@@ -86,9 +87,13 @@ def properties(self) -> dict:
             "user_agent_str": self.user_agent_str,
             "mode": self.mode,
         }
+
         if self.user_id is not None:
             props.update({"user_id": self.user_id, "organization_id": self.organization_id})
 
+        if get_config().anonymize_events:
```

Comment:
Is `$process_person_profile` defined in posthog docs? Could you link them?
Also, it not having it the same thing as setting it equal to `True` or does it absence mean something else? That is, I see no way of setting it to `True` so it's a bit confusing. If it's the same as `True` can we set it as that when we define `props` and then update it here.

Reply:
I see you've linked them in tests. I would include it in code. Either here, or where you define the `anonymize_events` in config.py.
It looks like not setting it or setting it to anything but False is equivalent to `True`. I would be explicit above.

Reply:
You could set this directly where props is defined as:
```
"$process_person_profile": not get_config().anonymize_events,
```

Reply:
I like this. 👍

Reply:
True is the default, hence we set it as false. I'll move it to the prop def as suggested.